adafruit__cpython.py

```diff
@@ -32,15 +32,17 @@
 
 **Hardware:**
 
-.. todo:: Add links to any specific hardware product page(s), or category page(s). Use unordered list & hyperlink rST
+.. todo:: Add links to any specific hardware product page(s), or category
```

Comment:
Your attempts to get this passing are admirable but RST doesn't play fair. You can undo this.